structure/saw.lua

```diff
@@ -2,6 +2,7 @@ Saw = Class {
     __includes=MeleeTower,
     init = function(self, gridOrigin, worldOrigin)
         self.towerType = "SAW"
+        self.cost = constants.STRUCTURE.SAW.COST
```

Comment:
Shouldn't obstacle have a cost too? What happens if you try and refund/delete one?